examples/sdkconfig.defaults

```diff
@@ -1,5 +1,5 @@
 # Rust often needs a bit of an extra main task stack size compared to C (the default is 3K)
-CONFIG_ESP_MAIN_TASK_STACK_SIZE=18576
+CONFIG_ESP_MAIN_TASK_STACK_SIZE=32000
```

Comment:
I would like to minimize the stack usage any chance we can reduce it closer the the original value>

Reply:
currently at 2400, at original value was experiencing stack overflows